default/stringtables/en.txt

```diff
@@ -1107,6 +1107,12 @@ Start server in hostless mode. Server accepts players in the multiplayer lobby,
 OPTIONS_DB_SKIP_CHECKSUM
 Skip comparing the resource directory checksums.  This allows faster startup when client and server are on the same machine, using the same resource directory.
 
+OPTIONS_DB_MPLOBBY_MAX_AI
+Restrict upper bound for number of AI could be in multiplayer game. Set to -1 for unbounded.
+
+OPTIONS_DB_MPLOBBY_MIN_HUMAN
+Restrict lower bound for number of human players could start the game.
```

Comment:
Set the minimum number of human players required to start a multiplayer game.

Reply:
Fixed.